### ISSUE TITLE
Contributions

### ISSUE THREAD

**Issue by lordtt13** [huggingface/notebooks#1]
I have some notebooks illustrating the use of transformers, may I make a PR?

**Comment by yjernite** [comment]
Hi @lordtt13 ! Could you share the notebooks or at least a quick description with us first so we can advise you on where would be the best way to share them?

**Comment by lordtt13** [comment]
Yes two of the examples are [a fine tune task](https://github.com/lordtt13/transformers-experiments/blob/master/Custom%20Tasks/fine-tune-classifier-script-loader.ipynb) and [a train from scratch task](https://github.com/lordtt13/transformers-experiments/blob/master/Custom%20Tasks/fine-tune-classifier-script-loader.ipynb). I also plan on adding some nb's where I'm showing both these tasks using TF versions of the models, as I feel TF is very underrepresented in exemplar scripts.
